tapiro/core/src/main/scala/io/buildo/tapiro/AkkaHttpMeta.scala

```diff
@@ -9,13 +9,12 @@ object AkkaHttpMeta {
     `package`: Term.Ref,
     imports: Set[Term.Ref],
     controllerName: Type.Name,
-    endpointsName: Term.Name,
+    httpEndpointsName: Term.Name,
     implicits: List[Term.Param],
     akkaHttpEndpoints: List[Defn.Val],
     routes: Term,
   ) => {
-    val tapirEndpoints = q"val endpoints = $endpointsName.create[AuthToken](statusCodes)"
-    val akkaHttpEndpointsName = Term.Name(s"${controllerName.syntax}AkkaHttpEndpoints")
+    val tapirEndpoints = q"val endpoints = $httpEndpointsName.create[AuthToken](statusCodes)"
```

Comment:
I was trying this out, I think here `httpEndpointsName` should be instead `tapirEndpointsName` (which must be added as as another parameter)

(and the same for http4s)